KerbalStuff/blueprints/api.py

```diff
@@ -545,73 +560,65 @@ def create_list():
 def create_mod():
     if not current_user.public:
         return { 'error': True, 'reason': 'Only users with public profiles may create mods.' }, 403
-    name = request.form.get('name')
-    game = request.form.get('game')
+    mod_name = request.form.get('name')
+    game_id = request.form.get('game') or request.form.get('game-id')
+    game_short = request.form.get('game-short-name')
     short_description = request.form.get('short-description')
-    version = request.form.get('version')
+    friendly_version = secure_filename(request.form.get('version', ''))
     game_version = request.form.get('game-version')
-    license = request.form.get('license')
-    ckan = request.form.get('ckan')
+    mod_licence = request.form.get('license')
```

Comment:
I don't understand the reason for renaming this field, but I think (personal preference) if both form fields and the variable matched their spellings (e.g. `mod_licence` -> `mod_license`). Not critical, just aesthetic.

Reply:
The foremost reason is readability -- we should always try to write code that could be read and understood as a plain english by _another guy_ who would come after us. Thus more descriptive names.
Second, `license` is a built-in constant: https://docs.python.org/3.7/library/constants.html#license

Reply:
As for the request attribute `license` -- changing it would break the API. Which would hit 3d-party  clients that rely on it.

Reply:
It's not a breaking change, just saying that you changed the spelling of `license`, it's just aesthetic :P